logical/src/main/java/org/apache/drill/common/logical/StoragePluginConfig.java

```diff
@@ -105,8 +106,8 @@ public enum AuthMode {
      */
     USER_TRANSLATION;
 
-    public static AuthMode parseOrDefault(String authMode) {
-      return !Strings.isNullOrEmpty(authMode) ? AuthMode.valueOf(authMode.toUpperCase()) : DRILL_PROCESS;
+    public static AuthMode parseOrDefault(String authMode, AuthMode defavlt) {
```

Comment:
I think you meant `default` rather than `defavlt`

Reply:
@cgivre I did wonder if that variable name would get a reaction. `default` is a reserved word in Java so this is a little Roman alphabet hack like writing `Class clazz;`.

Reply:
lol... Ok... well in that case, +1 (pending CI).  Thank you for turning this around so quickly.